jkube-kit/enricher/specific/src/test/java/org/eclipse/jkube/kit/enricher/specific/ServiceDiscoveryEnricherTest.java

```diff
@@ -69,51 +71,38 @@ public void setUp() {
     }
 
     @Test
-    public void testDiscoveryLabel() {
+    void discoveryLabel() {
         enricher.addAnnotations(builder);
         String value = builder.buildMetadata().getLabels().get("discovery.3scale.net");
         assertThat(value).isEqualTo(discoverable);
     }
 
     @Test
-    public void testDescriptionPathAnnotation() {
-        enricher.addAnnotations(builder);
-        assertAnnotation(descriptionPath, "description-path");
-    }
-
-    @Test
-    public void testDiscoveryVersionAnnotation() {
-        enricher.addAnnotations(builder);
-        assertAnnotation(discoveryVersion, "discovery-version");
-    }
-
-    @Test
-    public void testDefaultDiscoveryVersionAnnotation() {
+    void defaultDiscoveryVersionAnnotation() {
         context.getConfiguration().getProcessorConfig().getConfig().get("jkube-service-discovery").remove("discoveryVersion");
         enricher.addAnnotations(builder);
         assertAnnotation("v1", "discovery-version");
     }
 
-    @Test
-    public void testPathAnnotation() {
-        enricher.addAnnotations(builder);
-        assertAnnotation("/rest-3scale", "path");
-    }
-
-    @Test
-    public void testPortAnnotation() {
+    @DisplayName("add annotations to service")
+    @ParameterizedTest(name = "''{0}'' annotation should be ''{1}''")
+    @MethodSource("annotations")
+    void addAnnotations(String annotation, String expectedValue) {
         enricher.addAnnotations(builder);
-        assertAnnotation(port, "port");
+        assertAnnotation(expectedValue, annotation);
```

Comment:
This doesn't look like the right way to use parameterized test. You can simply assert map entries instead.